dag_in_context/Cargo.lock

```diff
@@ -152,7 +152,7 @@ dependencies = [
 [[package]]
 name = "bril-rs"
 version = "0.1.0"
-source = "git+https://github.com/uwplse/bril?rev=78881c45aa53231915f333d1d6dcc26cedc63b57#78881c45aa53231915f333d1d6dcc26cedc63b57"
+source = "git+https://github.com/uwplse/bril?rev=e2be3f5#e2be3f5d7e160f02b7aed0ef2bcc3e13ae722d2b"
```

Comment:
Why is there a change to the lock but not the toml?

Reply:
Not sure, I think this happened when I rebased on main.

Reply:
I'm pretty sure it's not safe to change one without the other. But looking at `main`, looks like this change is already there: https://github.com/egraphs-good/eggcc/blob/main/Cargo.lock#L199 so you should be okay to discard this?

Reply:
See Yihong's thread on Slack?
I can't discard this change because this file gets rebuilt by Rust.